### ISSUE TITLE
Localization support

### ISSUE THREAD

**Issue by amoraitis** [amoraitis/TodoList#20]
#### Anyone can improve the existing translations, feel free to extend.

**Is your feature request related to a problem? Please describe.**

We really should create a multilingual website. Thus an app like this will reach a wider audience

**Describe the solution you'd like**

We could definitely use localization as it is described in [the docs](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/localization?view=aspnetcore-2.1)
Using .resx files is a safe solution.

**Describe alternatives you've considered**
We could also use transifex

**Comment by MetzeloS** [comment]
Just made it possible to add localization to the app.
Temp change of the language can happen by adding "culture" attribute in the url
eg. http://localhost:47818/Todos/Home?culture=el-gr
Currently there is only 1 entry for English and Greek for testing purposes.
In order to extend it, we need to add new entries in the .resx files and then include them in the View files.

**Comment by amoraitis** [comment]
> Just made it possible to add localization to the app.
> Temp change of the language can happen by adding "culture" attribute in the url
> eg. http://localhost:47818/Todos/Home?culture=el-gr
> Currently there is only 1 entry for English and Greek for testing purposes.
> In order to extend it, we need to add new entries in the .resx files and then include them in the View files.

Your contirbution is just awesome, we can keep this issue open and make a list of languages we could add in the future.

**Comment by MetzeloS** [comment]
I think I forked after your namespace changes etc.
Please make sure I am working on a correct forked version because I am a
newbie with GIT.
I also created an upstream for your repo in order to pull any changes in
the future.
https://github.com/MetzeloS/TodoList/commits/develop

**Comment by amoraitis** [comment]
@MetzeloS yes you have the latest version, I merged your PR. I haven't noticed it the time I authored the comment. Thus I edited it.

**Comment by karandesai28** [comment]
Hello,
I have done translation of English to Hindi and I want to send Pull Request. Before that I want to see the preview, however localization isn't working on localhost. I tried appending "?culture=el-gr" to routes also but it kept showing English version only. Can any one please specify of what I could be missing?

**Comment by amoraitis** [comment]
> Hello,
> I have done translation of English to Hindi and I want to send Pull Request. Before that I want to see the preview, however localization isn't working on localhost. I tried appending "?culture=el-gr" to routes also but it kept showing English version only. Can any one please specify of what I could be missing?

Sorry for the late reply(I had a problem with my pc), let me look into that.

**Comment by amoraitis** [comment]
@karandesai28 Actually, localization is not widely used currently. You can add yours, see [this](https://github.com/amoraitis/TodoList/blob/c401f9c115188a5de5a742bb13d0fbf6975289da/TodoList.Web/Views/Todos/Home.cshtml#L7). @MetzeloS referenced 1 localization string because it was the only one by that time(Its the title of the Home page). I will open a new issue for referencing localization strings instead of text.

Feel free to open your PR. Your changes will be merged.

**Comment by karandesai28** [comment]
> @karandesai28 Actually, localization is not widely used currently. You can add yours, see [this](https://github.com/amoraitis/TodoList/blob/c401f9c115188a5de5a742bb13d0fbf6975289da/TodoList.Web/Views/Todos/Home.cshtml#L7). @MetzeloS referenced 1 localization string because it was the only one by that time(Its the title of the Home page). I will open a new issue for referencing localization strings instead of text.
> 
> Feel free to open your PR. Your changes will be merged.

Thank You. I didn't notice that. I have sent the pull request (https://github.com/amoraitis/TodoList/pull/60)

**Comment by TaarnStar** [comment]
Hi, I would like to add the localization for Italian and Croatian. Can I just fork the project or I need to do something before contributing? Thanks :)

**Comment by amoraitis** [comment]
Hi @TaarnStar, please go ahead and fork.🎊

**Comment by mhrnnoi** [comment]
Hi there! I noticed this issue and would be interested in contributing to the project by adding a Persian language translation. Is this issue still open, and would it be possible to add a Persian translation? I'd be happy to create a new .resx file for the Persian language. Please let me know if this would be helpful, and if there are any other guidelines or instructions I should follow. Thank you!"
